Tests/Api/ClientTest.php

```diff
@@ -0,0 +1,312 @@
+<?php
+
+namespace Test\Werkspot\BingAdsApiBundle\Api;
+
+use BingAds\Bulk\ReportTimePeriod;
+use BingAds\Proxy\ClientProxy;
+use Mockery;
+use Symfony\Component\Filesystem\Filesystem;
+use Werkspot\BingAdsApiBundle\Api\Client;
+use Werkspot\BingAdsApiBundle\Api\Exceptions;
+use Werkspot\BingAdsApiBundle\Guzzle\RequestNewAccessToken;
+use Werkspot\BingAdsApiBundle\Api\Helper;
+
+class ClientTest extends \PHPUnit_Framework_TestCase
+{
+    private $accessToken = "2ec09aeccaf634d982eec793037e37fe";
+    private $refreshToken = "0c59f7e609b0cc467067e39d523116ce";
+
+    public function testGeoLocationPerformanceReportReturnsArrayWithCsv()
+    {
+        $apiClient = $this->getApiClient(
+            $this->getRequestNewAccessTokenMock(),
+            $this->getClientProxyMock(),
+            $this->getFileHelperMock(),
+            $this->getCsvHelperMock(),
+            $this->getTimeHelperMock()
+        );
+        $result = $apiClient->get(['TimePeriod', 'AccountName', 'AdGroupId'], 'GeoLocationPerformanceReport', ReportTimePeriod::LastWeek);
```

Comment:
the only option is GeoLocationPerformanceReport currently, so why are we making it possible to pass the name?